include/ceed-impl.h

```diff
@@ -387,4 +388,7 @@ struct CeedOperator_private {
   CeedContextFieldLabel *context_labels;
 };
 
+CEED_INTERN int CeedOperatorGetFallback(CeedOperator op,
```

Comment:
I don't want this called by a backend. We don't have any functions in here, but I'm thinking of moving all the fallback related functions to this header file.